crates/precompile/Cargo.toml

```diff
@@ -98,6 +103,8 @@ secp256r1 = ["dep:p256"]
 
 # Enables the KZG point evaluation precompile.
 c-kzg = ["dep:c-kzg", "revm-primitives/c-kzg"]
+kzg-rs = ["dep:kzg-rs", "revm-primitives/kzg-rs"]
```

Comment:
> One more thing, feature should be added in revm-precompile
Are you referring to this? I believe that I already included it

Reply:
Not this, check cancun function in iirc lib.rs file. There is recent commit (second or third commit) on main that touches this part. 

There was a condition if ckzg is not enabled to not add precompile.